tests/plugins/test_settings.py

```diff
@@ -138,3 +176,277 @@ def test_load_plugin_config_with_env_var(
+    )
+
+    assert not err
+    assert section_banner not in out
+
+
+def test_conda_config_describe_unknown_plugin_setting(
+    condarc_plugin_manager, conda_cli
+):
+    """
+    Ensure that the correct error message is displayed when an unknown plugin setting is used
+    """
+    with pytest.raises(
+        ArgumentError, match="Invalid configuration parameters: \n  - invalid_setting"
+    ):
+        conda_cli("config", "--describe", "plugins.invalid_setting")
+
+
+def test_conda_config_show_includes_plugin_settings(
+    monkeypatch: MonkeyPatch, condarc_plugin_manager, conda_cli, tmp_path
+):
+    """
+    Ensure that the show command includes plugin settings
+    """
+    condarc = tmp_path / "condarc"
+    monkeypatch.setenv("CONDARC", str(condarc))
+    out, err, _ = conda_cli(
+        "config",
+        "--file",
+        condarc,
+        "--set",
+        f"plugins.{STRING_PARAMETER_NAME}",
+        "value_one",
+    )
+
+    out, err, _ = conda_cli("config", "--show")
+    config_data = YAML().load(out)
+
+    assert not err
+    assert config_data["plugins"][STRING_PARAMETER_NAME] == "value_one"
+    assert config_data["plugins"][SEQ_PARAMETER_NAME] == []
+    assert config_data["plugins"][MAP_PARAMETER_NAME] == {}
+
+    out, err, _ = conda_cli("config", "--show", "--json")
+
+    config_data = json.loads(out)
+    assert config_data["plugins"][STRING_PARAMETER_NAME] == "value_one"
+    assert config_data["plugins"][SEQ_PARAMETER_NAME] == []
+    assert config_data["plugins"][MAP_PARAMETER_NAME] == {}
+
+
+@pytest.mark.parametrize(
+    "parameter_name, expected_output",
+    [
+        (STRING_PARAMETER_NAME, "plugins:\n  string_parameter: value_one\n"),
+        (SEQ_PARAMETER_NAME, "plugins:\n  seq_parameter: []\n"),
+        (MAP_PARAMETER_NAME, "plugins:\n  map_parameter: {}\n"),
+        (
+            "non_existent_parameter",
+            ArgumentError(
+                "Invalid configuration parameters: \n  - non_existent_parameter"
+            ),
+        ),
+    ],
+)
+def test_conda_config_show_for_individual_settings(
+    monkeypatch: MonkeyPatch,
+    condarc_plugin_manager,
+    conda_cli,
+    tmp_path,
+    parameter_name,
+    expected_output,
+):
+    """
+    Ensure that the show command includes plugin settings
+    """
+    condarc = tmp_path / "condarc"
+    monkeypatch.setenv("CONDARC", str(condarc))
+    out, err, _ = conda_cli(
+        "config",
+        "--file",
+        condarc,
+        "--set",
+        f"plugins.{STRING_PARAMETER_NAME}",
+        "value_one",
+    )
+
+    if isinstance(expected_output, Exception):
+        with pytest.raises(ArgumentError, match=expected_output.message):
+            out, err, _ = conda_cli("config", "--show", f"plugins.{parameter_name}")
+    else:
+        out, err, _ = conda_cli("config", "--show", f"plugins.{parameter_name}")
+        assert not err
+        assert out == expected_output
```

Comment:
There are existing patterns in our test suite of using `contextlib.nullcontext` for this kind of testing:

```suggestion
    with pytest.raises(ArgumentError, match=expected_output.message) if isinstance(expected_output, Exception) else nullcontext():
        out, err, _ = conda_cli("config", "--show", f"plugins.{parameter_name}")
        assert not err
        assert out == expected_output
```